typings/index.d.ts

```diff
@@ -5463,8 +5460,6 @@ export interface ShardingManagerOptions {
   execArgv?: string[];
 }
 
-export { Snowflake };
-
```

Comment:
Please don’t add miscellaneous changes to your PR

Reply:
This line is also moved to line 648

Reply:
No clue what that is but it doesn’t sound relevant to this PR

Reply:
reverted